src/components/tl/cuda/tl_cuda.h

```diff
@@ -151,12 +156,12 @@ typedef struct ucc_tl_cuda_task {
     ucc_tl_cuda_shm_barrier_t *bar;
     union {
         struct {
-            int                     stage;
-            ucc_tl_cuda_mem_info_t  mem_info_src;
-            ucc_tl_cuda_mem_info_t  mem_info_dst;
-            void                   *peer_map_addr_src[UCC_TL_CUDA_MAX_PEERS];
-            void                   *peer_map_addr_dst[UCC_TL_CUDA_MAX_PEERS];
-            void                   *copy_done;
+            int                    stage;
+            ucc_tl_cuda_mem_info_t mem_info_src;
+            ucc_tl_cuda_mem_info_t mem_info_dst;
+            void                  *peer_map_addr_src[UCC_TL_CUDA_MAX_PEERS];
+            void                  *peer_map_addr_dst[UCC_TL_CUDA_MAX_PEERS];
+            void                  *copy_done;
         } alltoall_ce;
```

Comment:
minor, i understand that alltoall_ce and alltoallv_ce algorithms will probably be identical, but anyway i think it's better to have a different struct for alltoallv

Reply:
done